README.md

```diff
@@ -29,6 +30,31 @@ Minimal supported `Gradle` version: `6.4`.
 * Customizable filters for instrumented classes
 
 ## Quickstart
+### Using IntelliJ coverage engine
```

Comment:
I'd suggest moving it to the very bottom as it's not really relevant for quickstart and should not bother 95% of our users

Reply:
OK